chromeos.kernelci.org

```diff
@@ -110,6 +110,30 @@ cmd_docker() {
         cros-tast \
         cros-sdk
     cd -
+
+    echo "Updating Docker images"
+    # Build the images with kci_docker
+    cd checkout/kernelci-core/config/docker
+    core_rev=$(git show --pretty=format:%H -s origin/chromeos.kernelci.org)
+    rev_arg="--build-arg core_rev=$core_rev"
+    px_arg='--prefix=kernelci/staging-'
+    args="build --push $px_arg $rev_arg"
+
+    # KernelCI tools
+    ./kci_docker $args cros-kernelci
+    ./kci_docker $args k8s $rev_arg --fragment=cros-kernelci
+
+    # Compiler toolchains
+    for clang in clang-11 clang-14 clang-16; do
+	./kci_docker $args $clang --fragment=kselftest --fragment=cros-kernelci
```

Comment:
Why is there a `cros-kernelci` fragment?  The `kernelci` fragment should work just fine with the `core_rev` set to the revision from the `chromeos.kernelci.org` branch.

Reply:
To make different name for docker images tag, that means they contain "cros" version of kernelci-core tools, so they will not overwrite existing images.

Reply:
Using a different fragment name just for that seems like a quick hack.  Using a prefix which can be used like for staging images would seem like a much cleaner approach.